packages/salesforcedx-vscode-core/package.json

```diff
@@ -25,7 +25,7 @@
     "Other"
   ],
   "dependencies": {
-    "@heroku/functions-core": "^0.4.0",
+    "@heroku/functions-core": "0.4.2",
     "@octokit/plugin-paginate-rest": "2.21.3",
```

Comment:
So on review of this flow I realized it's probably not a good thing to have ^ deps in the dependancies list and not when we do the packaging so I just pinned both to 4.0.2 which was the current version.